frontend/app/src/scenes/Home/scenes/Category/scenes/subcategories/scenes/interfaces.ts

```diff
@@ -0,0 +1,30 @@
+export interface ItemStateProps {
+  items: Item[];
+  isLoading: boolean;
+  pagination: {
+    current: number,
+    pageSize: number,
+    total: number,
+  };
+}
+
+export interface Item {
```

Comment:
The item interface is already defined in home. Use the same.

Reply:
ah yes.